README.md

```diff
@@ -27,7 +27,26 @@ description: Control the splash screen for your app.
 
 # cordova-plugin-splashscreen
 
-This plugin is required to work with splash screens. This plugin displays and hides a splash screen during application launch.
+This plugin displays and hides a splash screen while your web application is launching. Using its methods you can also show and hide the splash screen manually.
+
+Note that this plugin is not about native loading screens, like e.g. ["Launch Screens" of iOS](https://developer.apple.com/design/human-interface-guidelines/ios/icons-and-images/launch-screen/). This plugin's splash screens are shown _after_ the native app has already started.
+
+- [Installation](#installation)
+- [Supported Platforms](#supported-platforms)
```

Comment:
Shouldn't we be consistent wit upper/lowercase in the heading?

Reply:
Could be, feel free to provide suggestions via the "Insert a suggestion" feature in the comment functionality here where stuff should be changed.

Reply:
Done for the headlines that are listed in the ToC